src/content/docs/en/pages/observe-journey/data-stream/edit-stream/configure-main-settings.mdx

```diff
@@ -17,6 +17,10 @@ While creating or editing a stream, there are a few specific main configurations
 
 To find more detailed information on the product and each configuration, see the [reference documentation](/en/documentation/products/observe/data-stream/).
 
+:::note
+Users who don't have the `Edit Data Stream` permission will be able to see the settings of the stream, but not to edit or create one. Also, the values in connector fields which are defined as `keys` won't be visible unless you have the `Edit Data Stream` permission.
```

Comment:
Users without the `Edit Data Stream` permission can only view the settings of the existing stream but not edit or create new ones. Values in connector fields defined as `keys` are only visible to users with the `Edit Data Stream` permission.